### PR TITLE
Add photo for Erwin

### PR DESCRIPTION
**Pull request by meerkatboy**

### CODE REVIEW

**Comment by realqijun** [comment]
LGMT
 /\_/\
( o.o )
 > ^ <
 😇

**Review by blanklogic — CHANGES_REQUESTED** [review]
Great photo of yourself but it appears the filename of your photo is meerkatboy.png.jpg and not meerkatboy.png, do consider making these changes.

**Review by blanklogic — APPROVED** [review]
Great photo and appreciate the changes. LGTM!

File: docs/images/meerkatboy.png
Comment (realqijun) [inline docs/images/meerkatboy.png:1]: Good looking

**Review by realqijun — COMMENTED** [review]
Good changes overall

**Pull request merged**